clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/deploy/ops/AbstractEurekaSupport.groovy

```diff
@@ -189,6 +192,11 @@ abstract class AbstractEurekaSupport {
         AbstractEurekaSupport.log.info("[$phaseName] - Failed marking discovery $discoveryStatus.value for instances ${errors}")
       }
     }
+    if (skipped) {
```

Comment:
nit: i would prefer something like 
```suggestion
    if (!skipped.isEmpty()) {
```
to make it less groovy and slightly more obvious what's going on